### PR TITLE
Update consensus sections

### PR DESCRIPTION
**Pull request by csarven**
See also https://github.com/solid/solid-wg-charter/issues/2

### CODE REVIEW

File: charter/index.html [hunk 1]
```diff
@@ -546,8 +546,12 @@ <h2>
 reopened at the discretion of the Chairs or the Director.
             </p>
             <p>
-                This charter is written in accordance with the <a href="https://www.w3.org/Consortium/Process/policies#Votes">W3C Process Document (Section 3.4, Votes)</a>, and includes no voting procedures beyond what the Process Document requires.
+                This charter is written in accordance with the <a href="https://www.w3.org/Consortium/Process/policies#Votes">W3C Process Document (Section 5.2.3, Votes)</a>, and includes the following voting procedures:
             </p>
+
+            <ul>
+              <li>One vote per W3C Member</li>
+            </ul>
```
Comment (pchampin) [inline charter/index.html:554]: what is the purpose of this change, as this rule *is* included in the process already...

https://beta.w3.org/2021/Process-20211102/#Votes
> In order to vote to resolve a substantive issue, an individual must be a group [participant](https://beta.w3.org/2021/Process-20211102/#participant). **Each organization represented in the group must have at most one vote**, even when the organization is represented by several participants in the group (including Invited Experts).

(emphasis is mine)

Furthermore, the phrasing you propose seems to exclude invited experts (who are not necessarily affiliated with members) -- which I guess is not your intention.

Comment (csarven) [reply charter/index.html]: Indeed! I must've glitched thinking what's in the Process is not there and that it should be :) I've updated the PR to only correct the section numbers. Thanks!

**Review by elf-pavlik — APPROVED** [review]

**Review by pchampin — APPROVED** [review]
